Assets/Scripts/PlayerManager.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerManager : MonoBehaviour
+{
+    #region Singleton
```

Comment:
Could you switch to David's Singleton base class for this logic?

Reply:
didn't see there is already one - switched